topics/sequence-analysis/tutorials/mapping/ref_genome_explanation.md

```diff
@@ -0,0 +1,14 @@
+We need a reference genome on which we map the reads.
+
+> ### {% icon question %} Questions
+>
+> 1. What is a reference genome?
+> 2. For human, several possible reference genomes are available: hg18, hg19, hg38, etc. What do they correspond to?
+> 2. Which reference genome should we use?
+>
+> > ### {% icon solution %} Solution
+> > 1. A reference genome (or reference assembly) is a DNA sequence assembled as a representative example of a species' DNA sequence. As they are often assembled from the sequencing of the DNA from different donors, they do not accurately represent the set of genes of any single person but a mosaic of different DNA sequences from each donor.
```

Comment:
Trying to be as general as possible (e.g. considering RNA viruses):

s/is a DNA sequence/is a set of nucleic acid sequences/
s/a species' DNA sequence./a species' genetic material./
s/from the sequencing of the DNA from different donors/from the sequencing of different individuals/
s/person/organism/
s/different DNA sequences from each donor./different nucleic acid sequences from each individual./